converter/converter.py

```diff
@@ -175,17 +175,19 @@ def save_configs(self, target_dir: str):
         Save the configs as text files in the target_dir.
         The files are: beam.dat, mat.dat, detect.dat and geo.dat.
         """
-        with open(path.join(target_dir, 'beam.dat'), 'w') as beam_f:
-            beam_f.write(self.beam_config)
+        target_dir = path.abspath(target_dir)
 
-        with open(path.join(target_dir, 'mat.dat'), 'w') as mat_f:
-            mat_f.write(self.mat_config)
+        with open(path.join(target_dir, 'beam.dat'), 'x') as beam_f:
```

Comment:
why `x` ?

Reply:
If we are gonna support multiple conversions running at the same time, we would like avoid overwriting any files that are still in use in case of two conversions happening in the same directory.